flowzone.yml

```diff
@@ -3362,7 +3371,9 @@ jobs:
           permissions: >-
             {
               "metadata": "read",
-              "contents": "read"
+              "contents": "read",
+              "issues": "read",
+              "pull_requests": "write"
```

Comment:
It might be worth adding comments about why particular write permissions are needed